webapps/api/backend/views/vpools.py

```diff
@@ -251,3 +251,24 @@ def create_hprm_config_files(self, local_storagerouter, vpool, parameters):
         return VPoolController.create_hprm_config_files.delay(parameters=parameters,
                                                               vpool_guid=vpool.guid,
                                                               local_storagerouter_guid=local_storagerouter.guid)
+
+    @action()
+    @log()
+    @required_roles(['read', 'write'])
+    @return_task()
+    @load(VPool)
+    def move_multiple(self, vdisks, target_storagerouter_guid, force=False):
```

Comment:
Change to: `def move_multiple(self, vpool, vdisk_guids, target_storagerouter_guid, force=False):`
The first parameter is initialized using the route param (/vpools/<vpool_guid>/move_multiple). The load decorator will then load a vPool object (the object type given to the load param) using the route param (in this case the vpool_guid)
All additional information comes in as a JSON serialized object with a key 'queryparams'. All keys within this queryparams represent the other params.
In our example, we would need to send:
```
{
    'queryparams': {'vdisk_guids': [<guid1>, ..],
                              'target_storagerouter_guid': <str_guid>}
}
```